atlassian/bitbucket/__init__.py

```diff
@@ -1437,7 +1437,7 @@ def add_pull_request_comment(
     def _url_pull_request_comment(
         self, project_key, repository_slug, pull_request_id, comment_id
     ):
-        url = "{}/{}".format(
+        url = "{}/comments/{}".format(
             self._url_pull_request(project_key, repository_slug, pull_request_id),
```

Comment:
Instead of adding `comments` the `_url_pull_request`should be replaced with `_url_pull_request_comments`. I've added this fix also to #622 .
```suggestion
        url = "{}/{}".format(
            self._url_pull_request_comments(project_key, repository_slug, pull_request_id),
```